tests/build.go

```diff
@@ -42,11 +42,7 @@ var buildTests = []func(t *testing.T, sb integration.Sandbox){
 	testBuildDetailsLink,
 	testBuildProgress,
 	testBuildAnnotations,
-	testBuildBuildArgNoKey,
-	testBuildLabelNoKey,
-	testBuildCacheExportNotSupported,
-	testBuildOCIExportNotSupported,
-	testBuildMultiPlatformNotSupported,
```

Comment:
If we want test tables I think we should group `testBuildBuildArgNoKey`, `testBuildLabelNoKey` and name it `testBuildKeyPairNoKey` and for `testBuildCacheExportNotSupported`, `testBuildOCIExportNotSupported`, `testBuildMultiPlatformNotSupported` have another one named `testBuildDockerNotSupported`.

Reply:
Ah missed your comment https://github.com/docker/buildx/pull/1998#discussion_r1322757861. So I think we should split them because they don't have the same input and purpose.